medicines/web/components/search-results/index.tsx

```diff
@@ -221,8 +221,8 @@ const SearchResults = (props: {
           )}
           <p className="ema-message">
             If the product information you are seeking does not appear below, it
-            is possible that the product holds a European licence and
-            its information may be available at the {emaWebsiteLink()} website.
```

Comment:
`tslint` made me move "its".

Reply:
it's ?

Reply:
@callerc1 I'm pretty sure "its" is correct.

"it's" is a contraction of "it is", which would mean the sentence is saying "…the product holds a European licence and _it is_ information may be available…"

Reply:
cool 👍